api/src/db/schema.ts

```diff
@@ -175,3 +175,61 @@ export const aiRequestLogs = sqliteTable("ai_request_logs", {
   log: text("log", { mode: "json" }).notNull(),
   createdAt: timestamps.createdAt,
 });
+
+export const collections = sqliteTable("collections", {
+  id: integer("id", { mode: "number" }).primaryKey({ autoIncrement: true }),
+  name: text("name").notNull().unique(),
+  createdAt: text("created_at").notNull().default(sql`(CURRENT_TIMESTAMP)`),
+  updatedAt: text("updated_at").notNull().default(sql`(CURRENT_TIMESTAMP)`),
```

Comment:
would suggest using the "timetamps" helper since it'll be iso8601

```suggestion
  ...timestamps,
```